zk_toolbox/README.md

```diff
@@ -106,3 +106,46 @@ zk_inception server
 ```
 
 You can specify the chain you are running by providing `--chain <chain_name>` argument
+
+### Prover
+
+To run the prover, follow these steps:
+
+First, initialize the prover:
```

Comment:
could you specify, the requirements here? 

I suppose it'd be the only doc the users will read. 
You can write something short, that it requires gcs + it requires gpu and then have a link to the main readme.